app/services/trading_strategies.py

```
@@ -2672,6 +2742,680 @@ def _calculate_backtest_macd(self, closes: List[float]) -> Tuple[float, float, s
+                        "support_level": recent_lows,
+                        "breakout_threshold": recent_lows * 0.99,
+                        "current_price": latest_price,
+                        "held_quantity": held_qty,
+                        "available_cash": available_cash
+                    }
+                }
+        return None
+
+    def _generate_backtest_scalping_signal(
+        self,
+        symbol: str,
```

Ruff (0.14.0)

Unused method argument: `symbol`

(ARG002)